app/views/ops/_logs_selected.html.haml

```diff
@@ -2,22 +2,17 @@
 - log_depot_uri        = "#{record.log_file_depot.try(:uri) || "N/A"}"
 - last_time            = record.last_log_sync_on
 - last_collection_time = last_time.blank? ? _("Never") : format_timezone(last_time.to_time, Time.zone, "gtl")
+- rows = [{:cells => {:label => "Log Depot URI", :value => log_depot_uri}},
+          {:cells => {:label => "Last Log Collection", :value => last_collection_time}}]
+
+- if record.respond_to?(:last_log_sync_message)
+  - rows = [{:cells => {:label => "Log Depot URI", :value => log_depot_uri}},
+          {:cells => {:label => "Last Log Collection", :value => last_collection_time}},
+          {:cells => {:label => "Last Message", :value => record.last_log_sync_message}}]
 
 #selected_div
   - if @sb[:selected_server_id] != my_server.id && !@selected_server.try(:started?)
     %h2= _("Collect Logs")
-  %h3= _("Basic Info")
-  .form-horizontal
-    .form-group
-      %label.col-md-2.control-label= _("Log Depot URI")
-      .col-md-10
-        %p.form-control-static= h(log_depot_uri)
-    .form-group
-      %label.col-md-2.control-label= _("Last Log Collection")
-      .col-md-10
-        %p.form-control-static= h(last_collection_time)
-    - if record.respond_to?(:last_log_sync_message)
-      .form-group
-        %label.col-md-2.control-label= _("Last Message")
-        .col-md-10
-          %p.form-control-static= h(record.last_log_sync_message)
+  = react("MiqStructuredList", {:title   => "Basic Information",
+                                :rows    => rows,
+                                :mode    => "miq_summary"})
```

Comment:
a similar approach mentioned above can be followed here..

And if possible, maybe we can reduce the `ops_helper.rb` to avoid code repetition